sealer/cmd/delete.go

```diff
@@ -20,23 +20,52 @@ import (
 	"regexp"
 
 	"github.com/alibaba/sealer/apply"
+	"github.com/alibaba/sealer/common"
 	"github.com/alibaba/sealer/logger"
+	"github.com/alibaba/sealer/utils"
 	"github.com/spf13/cobra"
 )
 
+var deleteArgs *common.RunArgs
+var deleteClusterFile string
+
 // deleteCmd represents the delete command
 var deleteCmd = &cobra.Command{
-	Use:     "delete",
-	Short:   "delete a cluster",
-	Long:    `if provider is BARESERVER will delete kubernetes nodes, or if provider is ALI_CLOUD, will delete all the infra resources`,
-	Example: `sealer delete -f /root/.sealer/mycluster/Clusterfile [--force]`,
+	Use:   "delete",
+	Short: "delete a cluster",
+	Long:  `if provider is BARESERVER will delete kubernetes nodes or IPList;  if provider is ALI_CLOUD, will delete all the infra resources or count`,
+	Example: `
+delete to default cluster: 
+	sealer delete --masters x.x.x.x --nodes x.x.x.x
+	sealer delete --masters x.x.x.x-x.x.x.y --nodes x.x.x.x-x.x.x.y
+delete to cluster by cloud provider, just set the number of masters or nodes:
+	sealer delete --masters 2 --nodes 3
+specify the cluster name(If there is only one cluster in the $HOME/.sealer directory, it should be applied. ):
+	sealer delete --masters 2 --nodes 3 -f /root/.sealer/specify-cluster/Clusterfile
+delete all:
+	sealer delete --all [--force]
+	sealer delete -f /root/.sealer/mycluster/Clusterfile [--force]
+`,
 	Run: func(cmd *cobra.Command, args []string) {
 		force, err := cmd.Flags().GetBool("force")
 		if err != nil {
 			logger.Error(err)
 			os.Exit(1)
```

Comment:
too many os.Exit in this function, I think you can put all the logic into one function and return error. So Run will much simpler, just needs one os.Exit(1).